ide/textmate.lexer/apichanges.xml

```diff
@@ -64,6 +64,19 @@
             </p>
         </description>
     </change>
+    <change id="injection_grammars" >
+        <api name="TextmateLexer"/>
+        <summary>Support for injection grammars added.</summary>
+        <version major="1" minor="3"/>
+        <date day="29" month="4" year="2019"/>
+        <author login="dbalek"/>
+        <compatibility addition="yes"/>
+        <description>
+            <p>
+                Support for injection grammars has been added.
+            </p>
+        </description>
```

Comment:
Please provide a link to detailed description. I've noticed there is nice description of the usecase - try to link to it. `<a name/>` and `<a href/>` should do it.

Also list added `classes` - there seem to be some.

Reply:
Fixed.